x-pack/test_serverless/functional/test_suites/search/connectors/connectors_overview.ts

```diff
@@ -36,11 +36,11 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       await pageObjects.svlSearchConnectorsPage.connectorOverviewPage.expectConnectorOverviewPageComponentsToExist();
     });
 
-    // FLAKY: https://github.com/elastic/kibana/issues/203462
+    // FLAKY: https://github.com/elastic/kibana/issues/203462 testing
     describe.skip('create and configure connector', () => {
       it('create connector and confirm connector configuration page is loaded', async () => {
         await pageObjects.svlSearchConnectorsPage.connectorConfigurationPage.createConnector();
-        await pageObjects.svlSearchConnectorsPage.connectorConfigurationPage.editType('zoom');
+        await pageObjects.svlSearchConnectorsPage.connectorConfigurationPage.editType('box');
```

Comment:
has zoom connector been removed?

Reply:
Not really. The issue was that the new `EuiComboBox` component we use to choose a connector displays the connector options in a virtualized PopOver and it was not getting the elements that were not rendered at a glance. But we can have a better component test management with the ComboBoxService https://github.com/elastic/kibana/blob/f151548b8bb418b0c0c6d0b25826b7af79f95fb3/x-pack/test_serverless/functional/page_objects/svl_search_connectors_page.ts#L9